en_US/data-integration/rule-sql-events-and-fields.md

```diff
@@ -119,17 +119,18 @@ If you want emqx to publish the event message, you can configure it in the `emqx
 
 ### Event topic available for FROM clause
 
-| Event topic name              | Explanation          |
-| ----------------------------- | :------------------- |
-| $events/message\_delivered    | message delivery     |
-| $events/message\_acked        | message acknowledged |
-| $events/message\_dropped      | Message dropped      |
-| $events/client\_connected     | Connection complete  |
-| $events/client\_disconnected  | Disconnect           |
-| $events/client\_connack       | Connection ack       |
-| $events/client\_check_authz_complete | Authorization check complete |
-| $events/session\_subscribed   | Subscribe            |
-| $events/session\_unsubscribed | Unsubcribe           |
+| Event topic name                     | Explanation                     |
+|--------------------------------------|:--------------------------------|
+| $events/message\_delivered           | message delivery                |
```

Comment:
Maybe we could be consistent, and capitalize or lowercase the first letter of all explanation